examples/test-app/steps/Country/template.html

```diff
@@ -8,19 +8,19 @@
 {% set phase="ALPHA" %}
```

Comment:
I like the fact we now have the template, content and class in the same directory.